### PR TITLE
Update for 2.1 (parent + total missing required)

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/products-super-power/enrich-products-variants.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ If you are looking for clothing with the color "red" and the size "S", the red S
 
 ![Search results with color red and size S](../img/Products_ProductModel_Search3.png)
 
+:::tips
+If you would like to display only products in the grid, you just have to filter on an attribute always at the product level, for example its identifier ("SKU").
```
Comment (NolwennP) [inline content/md/products-super-power/enrich-products-variants.md]: `If you want` ?

Comment (NolwennP) [inline content/md/products-super-power/enrich-products-variants.md]: What does `always` mean here? I don't understand.

File: content/md/products-super-power/enrich-products-variants.md [hunk 2]
```diff
@@ -72,9 +85,13 @@ For more details about the product form, you can read [Enrich your product](/art
 
 ## Header and infos
 
-For products with variants, the number of variant products behind the model and the number of complete variant products for the model are displayed in the product form header (`Variant products`).
+For product models, the following properties are displayed in the product form header:
+- The number of complete variant products out of the number of variant products for a product model in `Variant products` (ex: 2/6 means that 2 variants out of 6 for the product model are complete)
+- The total `missing required attributes` (__since the 2.1__) for a product model. If you click on it, it will display only the attributes required for the completeness without value (filter on display `all missing required attributes`).
```
Comment (NolwennP) [inline content/md/products-super-power/enrich-products-variants.md]: `the attribute that are required for the completeness but that are still incomplete`

Comment (NolwennP) [reply content/md/products-super-power/enrich-products-variants.md]: Also, I don't understand what you wanna say by `(filter on display `all missing required attributes`)`

File: content/md/products-super-power/enrich-products-variants.md (hunk 1 above)
Comment (Delphineray) [reply content/md/products-super-power/enrich-products-variants.md]: ok changed

File: content/md/products-super-power/enrich-products-variants.md (hunk 2 above)
Comment (Delphineray) [reply content/md/products-super-power/enrich-products-variants.md]: ok

**Pull request merged**